cacerts/certs.go

```diff
@@ -45,12 +45,6 @@ const (
 	DefaultCAFile string = "/etc/ssl/certs/ca-certificates.crt"
 )
 
-// GenerateHashLinks generates symlinks the given directory point to the given certificates paths.
-// The name of each symlink file will be of the format HHHHHHHH.D where HHHHHHHH is the 8 character
-// hexidecimal representation of the SubjectNameHash. D shall be the integer '0' unless there is a hash
-// conflict in which case D shall be incremented for the latter of the conflicting certs.
-//
-// These links are used by openssl to lookup a given CA by subject name.
```

Comment:
I don't think we should delete these comments. I believe they are still applicable. Is there a reason you removed them? or just an accident?